src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = function(num) {
+    if (typeof num !== 'number') {
```

Comment:
just check for undefined 
```suggestion
    if (num === undefined) {
```